specification/metrics/pricingquantity.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Quantity
+
+The Pricing Quantity represents the volume of a given resource and/or service used or purchased based on the [Pricing Measure](#pricingmeasure). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not resource and service consumption.
```

Comment:
Does it make sense to include something that explains how Pricing Quantity * Pricing Unit = <insert cost column here>?